terraform/modules/rds-serverless/main.tf

```diff
@@ -0,0 +1,90 @@
+provider "aws" {
+  region = "ap-southeast-2"
+}
+
+######################################
+# Data sources to get VPC and subnets
+######################################
+# https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/vpc
+data "aws_vpc" "da-c02-vpc" {
+  #id = "${var.vpc_id}"
+  id = "vpc-02ec2836691642ace"
+}
+
+data "aws_subnet_ids" "da-c02-vpc" {
+  vpc_id = data.aws_vpc.da-c02-vpc.id
+}
+
+#############
+# RDS Aurora
+#############
+module "aurora" {
```

Comment:
Although it might work, I think its best if we just use the basic terraform (no external modules), at least for now.

The main thing will be a `resource "rds_cluster_instance" "wordpress"` (for example)

More info: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_cluster_instance